src/main/FXML/MainScreen.fxml

```diff
@@ -30,7 +30,7 @@
             <Font name="Consolas" size="33.0" />
          </font>
       </Text>
-      <Text fill="#ff1fc7" layoutX="63.0" layoutY="62.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Highest score achieved this session:" wrappingWidth="399.6708984375">
+      <Text fill="#ff1fc7" layoutX="63.0" layoutY="62.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Highest score:" wrappingWidth="399.6708984375">
```

Comment:
Can you also change the position of the text? Now we have a really big space between the text and the score. <review